reference/orders.v2.oas2.yml

```diff
@@ -5189,6 +5222,30 @@ components:
           shipping_quotes:
             url: 'https://api.bigcommerce.com/stores/{$$.env.store_hash}/v2/orders/163/consignments/shipping/64/shipping_quotes'
             resource: /orders/163/consignments/shipping/64/shipping_quotes
+    shippingConsignment_Put:
+      allOf:
+        - type: object
+          properties:
+            id:
+              type: integer
+              example: 99
+              description: ID of shipping consignment to update.
+          required:
+            - id
+        - $ref: '#/components/schemas/shippingConsignment_Base'
```

Comment:
@donald-nguyen-bc : added the shipping consignment base reference here.

Reply:
thanks Munjal 🙇